helm/prometheus-rules/templates/alerting-rules/certificate.all.rules.yml

```diff
@@ -45,14 +45,25 @@ spec:
         severity: page
         team: {{ include "providerTeam" . }}
         topic: cert-manager
-    - alert: ManagedCertificateCRWillExpireInLessThanTwoWeeks
+    - alert: GiantswarmManagedCertificateCRWillExpireInLessThanTwoWeeks
       annotations:
         description: '{{`Certificate CR {{ $labels.namespace }}/{{ $labels.name }} on {{ $labels.cluster_id }} will expire in less than two weeks.`}}'
         opsrecipe: managed-app-cert-manager/certificate-secret-will-expire-in-less-than-two-weeks/
-      expr: (cert_exporter_certificate_cr_not_after{managed_issuer="true"} - time()) < 2 * 7 * 24 * 60 * 60
+      expr: (cert_exporter_certificate_cr_not_after{managed_issuer="true",namespace=~"kube-system|giantswarm"} - time()) < 2 * 7 * 24 * 60 * 60
```

Comment:
For CAPI based management clusters there are also some certificates in `monitoring` and `default` namespace:

```
NAMESPACE     NAME                                      READY   SECRET                                            AGE
default       happaapi-certs                            True    happaapi-certs                                    67d
giantswarm    app-admission-controller-certificates     True    app-admission-controller-certificates             93d
giantswarm    athena-certs-secret                       True    athena-certs-secret                               67d
giantswarm    capa-serving-cert                         True    capa-webhook-service-cert                         93d
giantswarm    capi-kubeadm-bootstrap-serving-cert       True    capi-kubeadm-bootstrap-webhook-service-cert       93d
giantswarm    capi-kubeadm-control-plane-serving-cert   True    capi-kubeadm-control-plane-webhook-service-cert   93d
giantswarm    capi-serving-cert                         True    capi-webhook-service-cert                         93d
giantswarm    dex-k8s-authenticator-le                  True    dex-k8s-authenticator-le                          67d
giantswarm    dex-tls-le                                True    dex-tls-le                                        67d
giantswarm    happa-certs                               True    happa-certs                                       67d
kube-system   aws-pod-identity-webhook                  True    aws-pod-identity-webhook                          93d
kube-system   kiam-agent                                True    kiam-agent-tls                                    93d
kube-system   kiam-ca                                   True    kiam-ca-tls                                       93d
kube-system   kiam-server                               True    kiam-server-tls                                   93d
monitoring    alertmanager-tls                          True    alertmanager-tls                                  70d
monitoring    grafana-tls                               True    grafana-tls                                       93d
monitoring    prometheus-tls                            True    prometheus-tls                                    70d

```

Reply:
I'm ok with monitoring, but the `default` NS sounds dangerous

Reply:
actually you know what? those alerts should be firing for WCs only, so I added another filter to the query